go/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"REPO_URL/YOUR-USER-OR-ORG-NAME/YOUR-REPO-NAME/gildedrose"
+	"fmt"
+	"os"
+	"strconv"
+)
+
+func main() {
+	fmt.Println("OMGHAI!")
+
+	days := 2
+	var err error
+	if len(os.Args) > 1 {
+		days, err = strconv.Atoi(os.Args[1])
+		if err != nil {
+			fmt.Println(err.Error())
+			os.Exit(1)
+		}
+		days++
+	}
+
+	for day := 0; day < days; day++ {
+		fmt.Printf("-------- day %d --------\n", day)
+		fmt.Println("Name, SellIn, Quality")
+		for i := 0; i < len(items); i++ {
```

Comment:
I can't see where "items" comes from. Did you forget to copy them here from texttest_fixture.go?

Reply:
or probably I just don't understand go well enough.. can you put them in this file anyway?

Reply:
`items` is a global variable set in the file **texttestfixtures.go**. But I've been thinking that maybe it's better to roll back the changes, leaving the `texttest_fixtures.go` file as it was.